### DISCUSSION TITLE
Mac Mini M1 Ventura Mouse Settings Has No Effect

### DISCUSSION THREAD

**Discussion by stevesob** [linearmouse/linearmouse#579] [General]
No matter what pointer speed settings I choose using System Preferences or LinearMouse, the mouse curser speed never changes. This happens on the Apple Magic Mouse, a generic Bluetooth mouse, and a USB mouse. I have tried changing the setting in the terminal and this has no effect either.

I'm stuck with too fast on the non-apple mouse and too slow on the Magic mouse.

I feel like some background program has taken over my mouse settings, does anyone know how to cure this? Thanks in advance --Steve O.

**Comment by lujjjh** [comment]
Hey Steve,

Have you tried creating a fresh user account in the system settings to check if the problem persists after switching to the new user?
